lms/envs/aws.py

```diff
@@ -669,6 +669,7 @@
     FIELD_OVERRIDE_PROVIDERS += (
         'ccx.overrides.CustomCoursesForEdxOverrideProvider',
     )
+CCX_MAX_STUDENTS_ALLOWED = ENV_TOKENS.get('CCX_MAX_STUDENTS_ALLOWED', CCX_MAX_STUDENTS_ALLOWED)
```

Comment:
If we're going to improve the gradebook to have server side pagination,
ideally the performance will be good enough to kick this up to at least 500
or even 1000 - which frankly is reasonable for an on campus course. more
than 1000 seems ridiculous, though.

On Wed, Nov 4, 2015 at 12:07 PM, Peter Pinch <notifications@github.com>
wrote:

> In lms/envs/aws.py
> <https://github.com/edx/edx-platform/pull/10483#discussion_r43908184>:
>
> > @@ -669,6 +669,7 @@
> >      FIELD_OVERRIDE_PROVIDERS += (
> >          'ccx.overrides.CustomCoursesForEdxOverrideProvider',
> >      )
> > +CCX_MAX_STUDENTS_ALLOWED = ENV_TOKENS.get('CCX_MAX_STUDENTS_ALLOWED', CCX_MAX_STUDENTS_ALLOWED)
>
> I think whether or not to make this configurable needs to be edX's
> decision. I'm fine as long as it isn't more than 200.
>
> @sarina <https://github.com/sarina> let us know what you think or whose
> opinions we need to bring in.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/edx/edx-platform/pull/10483/files#r43908184>.
>